src/lib_dispatch.cpp

```diff
@@ -39,6 +39,14 @@ extern "C"
   void sgetri_(int *n, float *A, int *lda, int *ipiv, float *work, int *lwork,
                int *info);
 
+  void dpttrf_(int *n, double *D, double *E, int *info);
```

Comment:
This will cause a conflict once OpenBLAS adds these to their headers. A better solution is to guard with:
```
#ifndef dpttrf_
void dpttrf_(int *n, double *D, double *E, int *info);
#endif
```
That way we won't get hit with redefinition in the near future. Also, let's add a comment so we would know to clean it up when `dpttrf_` becomes part of the OpenBLAS release.